### PR TITLE
Overhaul gui skinning page for the new theme editor in 3.4

### PR DESCRIPTION
**Pull request by skyace65**
Overhauls the entire GUI skinning page for the new theme editor in 3.4. Stuff has been removed, moved around, and the tutorial has been removed since this is supposed to be a manual page.

The only thing this doesn't cover is what purpose custom theme items or custom item types serve because I do not know.

Also does anyone know if the warning about the custom theme option in project settings is still relevant?

### CODE REVIEW

File: tutorials/ui/gui_skinning.rst [hunk 1]
```diff
@@ -48,32 +23,125 @@ user interfaces.
    Don't use the custom theme option in the Project Settings, as there
    are known bugs with theme propagation. Instead, apply your theme to the
    root Control node's Theme property. It will propagate to instanced scenes
-   automatically. To get correct theming in the editor for instanced scenes,
+   automatically. To get correct themeing in the editor for instanced scenes,
```
Comment (mhilbrunner) [inline tutorials/ui/gui_skinning.rst:26]:
```suggestion
   automatically. To get correct theming in the editor for instanced scenes,
```

I think this spelling is actually correct (or at least more widely used), i.e. https://en.wikipedia.org/wiki/Theming

File: tutorials/ui/gui_skinning.rst [hunk 2]
```diff
@@ -48,32 +23,125 @@ user interfaces.
...
    you can apply the theme resource to the instanced scene's root node as well.
 
-Theme options
--------------
+Creating a theme
+----------------
 
-Each kind of option in a theme can be:
+Theme's can be created from any control node. Select a control node in the scene
```
Comment (mhilbrunner) [inline tutorials/ui/gui_skinning.rst:32]:
```suggestion
Themes can be created from any control node. Select a control node in the scene
```

File: tutorials/ui/gui_skinning.rst [hunk 3]
```diff
@@ -48,32 +23,125 @@ user interfaces.
...
+
+Theme items
+-----------
+
+In the theme editor, next to the default preview window, is where you make changes
+to your theme. Clicking the plus button opens the add item type menu.
+
+.. image:: img/add_item_type.png
+
+from here select the control node you want your theme to modify and click **Ok**. You
```
Comment (mhilbrunner) [inline tutorials/ui/gui_skinning.rst:56]:
```suggestion
From here select the control node you want your theme to modify and click **Ok**. You
```

File: tutorials/ui/gui_skinning.rst [hunk 4]
```diff
@@ -48,32 +23,125 @@ user interfaces.
...
+the default font the theme uses. That can only be customized in the inspector under
+the selected theme.
+
+.. image:: img/default_font.png
+
+Theme items
+-----------
+
+In the theme editor, next to the default preview window, is where you make changes
+to your theme. Clicking the plus button opens the add item type menu.
```
Comment (mhilbrunner) [inline tutorials/ui/gui_skinning.rst:52]: > Clicking the plus button opens the add item type menu.

This is hard to read, should add some formatting about the item menu text to distinguish it from the rest of the sentence.

**Comment by skyace65** [comment]
Fixed.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thanks!

**Comment by pycbouh** [comment]
Oi, I was actually in the middle of writing a new article to update for the changes and because the old one was pretty bad. Not sure what to do now...

**Comment by skyace65** [comment]
@pycbouh if you've got any information on how to use custom theme items or custom item types the page still needs that.

**Comment by pycbouh** [comment]
@skyace65 I was going for a completely different structure, better explaining where and why people would and should use themes, to accompany the editor PR itself. Your version is functional, in the sense that it describes parts of the process as they are, but that's not how I would approach the matter as this doesn't help the adoption of the feature (lack of which was the reason I remade the editor in the first place).

**Comment by pycbouh** [comment]
I think as a middle ground, and since your work is more about instructions, we can rework your article into more of a straight up editor tutorial and I can still write a new article on the purpose of themes and how to apply them, and we can crosslink them. I'll look into it.

**Pull request merged**